jaws-journey-deploy/orb.yml

```diff
@@ -537,8 +549,12 @@ jobs:
             - store_artifacts:
                 path: ~/test-results/junit
                 destination: ~/reports/diagrams
-            - store_artifacts:
-                path: ~/reports/jacoco
+            - when:
+                condition: << parameters.store_jacoco_artifacts >>
+                steps:
+                  - store_artifacts:
+                      name: store jacoco artifacts
+                      path: ~/reports/jacoco
```

Comment:
Is this the right path? everything else is using ./reports/jacoco - obviously this would be an existing bug.